### ISSUE TITLE
in the newest v0.11.0, function call breaks when using Claude LLM

### ISSUE THREAD

**Issue by zamia** [livekit/agents#1017]
After upgrade agent to v0.11, plugins-anthropic to v0.2.2, when using function call feature and using claude llm, there will be a 400 Bad Request error.

How to reproduce:

1. Using the example `function_calling_weather.py`
2. Change LLM to `claude-3-5-sonnet-20241022`
3. chat with agent.

(and openai llm works fine).

The problem is after function call, the 2nd call to LLM will be break. The log detail is here:

```
2024-10-31 10:46:45,680 - DEBUG anthropic._base_client - HTTP Request: POST https://api.anthropic.com/v1/messages "400 Bad Request" {"pid": 97601, "job_id": "AJ_9yyzzj3Bvwuj"}
2024-10-31 10:46:45,680 - DEBUG anthropic._base_client - Encountered httpx.HTTPStatusError
Traceback (most recent call last):
  File "/Users/medal/src/3rd/livekit-agents/examples/voice-pipeline-agent/.venv/lib/python3.12/site-packages/anthropic/_base_client.py", line 1612, in _request
    response.raise_for_status()
  File "/Users/medal/src/3rd/livekit-agents/examples/voice-pipeline-agent/.venv/lib/python3.12/site-packages/httpx/_models.py", line 763, in raise_for_status
    raise HTTPStatusError(message, request=request, response=self)
httpx.HTTPStatusError: Client error '400 Bad Request' for url 'https://api.anthropic.com/v1/messages'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400 {"pid": 97601, "job_id": "AJ_9yyzzj3Bvwuj"}
2024-10-31 10:46:45,685 - DEBUG anthropic._base_client - Not retrying as header `x-should-retry` is set to `false` {"pid": 97601, "job_id": "AJ_9yyzzj3Bvwuj"}
2024-10-31 10:46:45,686 - DEBUG anthropic._base_client - Re-raising status error {"pid": 97601, "job_id": "AJ_9yyzzj3Bvwuj"}
2024-10-31 10:46:45,689 - DEBUG livekit.agents.pipeline - speech playout finished {"speech_id": "23dbd39fdf93", "interrupted": false, "pid": 97601, "job_id": "AJ_9yyzzj3Bvwuj"}
```

**Comment by sulihuangl** [comment]
Same issue encountered

**Comment by theomonnom** [comment]
It should be fixed in #1018. Will release a new version soon
